test/lib/services/html_converter_test.rb

```diff
@@ -131,13 +131,100 @@ def test_transform_html_on_default_lang_with_query_pattern_and_supported_lang
+    end
+
+    def test_transform_html_on_default_lang_do_not_change_canonical_tag_query_pattern
+      dom = get_dom('<html><head><link rel="canonical" href="http://my-site.com/" /></head><body></body></html>')
+      settings = {
+        'default_lang' => 'en',
+        'supported_langs' => %w[en ja vi],
+        'url_pattern' => 'query',
+        'translate_canonical_tag' => true
+      }
+      store, headers = store_headers_factory(settings, 'http://my-site.com/')
+      url_lang_switcher = Wovnrb::UrlLanguageSwitcher.new(store)
+      converter = HtmlConverter.new(dom, store, headers, url_lang_switcher)
+      translated_html = converter.build
+
+      expected_html = "<html lang=\"en\"><head><meta http-equiv=\"Content-Type\" content=\"text/html; charset=UTF-8\"><script src=\"https://j.wovn.io/1\" async=\"true\" data-wovnio=\"key=123456&amp;backend=true&amp;currentLang=en&amp;defaultLang=en&amp;urlPattern=query&amp;langCodeAliases={}&amp;langParamName=wovn&amp;version=WOVN.rb_#{VERSION}\" data-wovnio-type=\"fallback_snippet\"></script><link rel=\"canonical\" href=\"http://my-site.com/\"><link rel=\"alternate\" hreflang=\"en\" href=\"http://my-site.com/\"><link rel=\"alternate\" hreflang=\"ja\" href=\"http://my-site.com/?wovn=ja\"><link rel=\"alternate\" hreflang=\"vi\" href=\"http://my-site.com/?wovn=vi\"></head><body></body></html>"
+      assert_equal(expected_html, translated_html)
+    end
+
+    def test_transform_html_on_default_lang_with_default_lang_alias
+      dom = get_dom('<html><head><link rel="canonical" href="http://my-site.com/" /></head><body></body></html>')
+      settings = {
+        'default_lang' => 'en',
+        'supported_langs' => %w[en ja vi],
+        'url_pattern' => 'query',
+        'translate_canonical_tag' => true,
+        'custom_lang_aliases' => { 'en' => 'english' }
+      }
+      store, headers = store_headers_factory(settings, 'http://my-site.com/')
+      url_lang_switcher = Wovnrb::UrlLanguageSwitcher.new(store)
+      converter = HtmlConverter.new(dom, store, headers, url_lang_switcher)
+      translated_html = converter.build
+
+      expected_html = '<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><script src="https://j.wovn.io/1" async="true" data-wovnio="key=123456&amp;backend=true&amp;currentLang=en&amp;defaultLang=en&amp;urlPattern=query&amp;langCodeAliases={&quot;en&quot;:&quot;english&quot;}&amp;langParamName=wovn&amp;version=WOVN.rb_3.5.0" data-wovnio-type="fallback_snippet"></script><link rel="canonical" href="http://my-site.com/?wovn=english"><link rel="alternate" hreflang="en" href="http://my-site.com/"><link rel="alternate" hreflang="ja" href="http://my-site.com/?wovn=ja"><link rel="alternate" hreflang="vi" href="http://my-site.com/?wovn=vi"></head><body></body></html>'
+      assert_equal(expected_html, translated_html)
+    end
+
+    def test_transform_html_on_target_lang_do_not_translate_canonical_tag
```

Comment:
test name is misleading. it really means that the feature is disabled. i have no problem changing tests to the `test 'description' do` syntax` since it can make it easier to read.

Reply:
I've updated the names they are more readable now.